src/pocketmine/plugin/PluginDescription.php

```diff
@@ -97,6 +100,7 @@ private function loadMap(array $plugin){
 
 		$this->api = array_map("\strval", (array) ($plugin["api"] ?? []));
 		$this->compatibleMcpeProtocols = array_map("\intval", (array) ($plugin["mcpe-protocol"] ?? []));
```

Comment:
Who added the `\` here? That's both wrong string literal syntax and unnecessary.

Reply:
It's not wrong and it's not unnecessary either, since imports aren't respected at this scope.

Reply:
apparently PhpStorm has a bug and this is not necessary, because callback references are evaluated in the root namespace scope anyway. However, it isn't technically wrong.

how about we just replace all callables with closures instead? lol...